### PR TITLE
Workspaces

### PR DESCRIPTION
**Pull request by lishaduck**
Deno is rolling out a new feature called workspaces.
It enables JSR monorepos to link to packages locally.
While `libs` is far less interconnected than `deno_std`, I figured that it might help if `@libs/testing` got some extra "tests" by having all the packages link to the local copy in development.

It should also enable a single dependency resolve and (probably) obsolete deno_config.ts, as I believe it automatically syncs formatting/linting/etc.

### CODE REVIEW

File: bundle/deno.jsonc
Comment (lowlighter) [inline bundle/deno.jsonc]: I don't think it's necessary to explicitely mention the `recommended` set in every conf, it's already the default

File: deno.jsonc
Comment (lowlighter) [inline deno.jsonc]: It makes vercel deployments fail with the following:

> error: Workspace member 'bundle/' has no deno.json file ('/vercel/path1/bundle/deno.json')
> Error: Build script failed with exit code 1

I think it's related to https://github.com/denoland/deno/issues/23343 and the fact that the [vercel-community/deno uses 1.42.1 currently](https://github.com/vercel-community/deno/blob/e8d1d546adda2c1a8e0c4c7765e358210168e9d7/src/deno-lambda.ts#L4). 

I know you can pass a `--version` flag to them according to their docs, but I'll prefer it to not do as it does not make sense to break lambdas just because of a conf change which is enterily not related to actual code.

I'll wait for a version bump on their side before merging this

**Review by lowlighter — CHANGES_REQUESTED** [review]
The `workspaces` setting seems blocked by [vercel-community/deno](https://github.com/vercel-community/deno)

Everything in `.vscode`/`.github` looks good to me so if you'd like to merge these changes now you could open a separate PR

As for the `lint.tags` I'd prefer keeping it less verbose since it's already the default

File: deno.jsonc
Comment (lishaduck) [reply deno.jsonc]: vercel-community/deno#155

**Review by lishaduck — COMMENTED** [review]
> As for the `lint.tags` I'd prefer keeping it less verbose since it's already the default 

Yeah, I hadn't realized it would make such a big diff when I made the change originally. It's reverted now.

**Comment by lishaduck** [comment]
Ok, it looks like it's not inheriting the fmt and lint configs. I'd assume that CI just needs to get bumped to 1.45.

**Comment by lishaduck** [comment]
> Ok, it looks like it's not inheriting the fmt and lint configs. I'd assume that CI just needs to get bumped to 1.45.

Hmm. CI should be using 1.45. Vercel deploys now, which is good. Hmm.

**Comment by lishaduck** [comment]
Superseded by #84.

**Pull request closed without merging**
